modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -302,12 +315,15 @@ class IECallContext
 
     const IEUnit                          &uu;
     cv::gimpl::GIslandExecutable::IOutput &out;
-    cv::gimpl::ie::SyncPrim               &sync;
 
     // NB: Need to gurantee that MediaFrame::View don't die until request is over.
     using Views = std::vector<std::unique_ptr<cv::MediaFrame::View>>;
     Views views;
 
+    // FIXME: Should be removed from here!!!
+    std::mutex m;
+    cv::GArg state;
```

Comment:
What is `state` and why it is `GArg` ?

Reply:
In fact it is `any`. 
The idea is simple, in case `Infer2` and `InferList` the `IECallContext` is shared between multiple `Callbacks` and sometimes it's needed to store some information like `cached_dims`, so on..

Reply:
Let's postpone it, removed.